source/collaborate/extend-mattermost-with-integrations.rst

```diff
@@ -1,7 +1,18 @@
 Extend Mattermost with integrations
 ===================================
 
-`Visit the Mattermost Marketplace <https://mattermost.com/marketplace/>`__ to find dozens of open source integrations to common tools like Jira, Jenkins, GitLab, with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
+`Visit the Mattermost Marketplace <https://mattermost.com/marketplace/>`__ to find open source integrations to common developer tools like `CircleCI <https://mattermost.com/marketplace/circleci/>`__, `Opsgenie <https://mattermost.com/marketplace/opsgenie/>`__, `PagerDuty Notifier <https://mattermost.com/marketplace/pagerduty/>`__, and `Zendesk App <https://mattermost.com/marketplace/zendesk-app/>`__; productivity tools like `Autolink <https://mattermost.com/marketplace/autolink-plugin/>`__, `ToDo <https://mattermost.com/marketplace/todo/>`__, and `WelcomeBot <https://mattermost.com/marketplace/welcomebot-plugin/>`__; as well as social tools like `Memes <https://mattermost.com/marketplace/memes-plugin/>`__ and `GIFs <https://mattermost.com/marketplace/giphy-plugin/>`__ that are freely available for use and customization.
+
+Pre-packaged integrations
```

Comment:
Missing Calls

Reply:
Also missing playbooks

Reply:
Chatted with @wiersgallak about this; Calls and Playbooks are considered product features rather than integrations.